feat(vue,nuxt): Add Vue and Nuxt SDK docs (#1710)
clerk/clerk-docs: docs/references/vue/use-session.mdx

---
title: useSession()
description: Access and manage the current user's session in your Vue application with Clerk's useSession() composable.
---

The `useSession()` composable provides access to the current user's [`Session`](/docs/references/javascript/session) object, as well as helpers for setting the active session.

## Returns

<Properties>
  - `isLoaded`
  - `Ref<boolean>`

  A boolean that indicates whether Clerk has completed initialization. Initially `false`, becomes `true` once Clerk loads.

  ---

  - `isSignedIn`
  - `Ref<boolean>`

>   A boolean that is set to `true` if the user is signed in.

Reviewer: ```suggestion
  A boolean that indicates whether a user is currently signed in.
```
